src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,27 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.13.0
+
+27 September 2023
+This version is compatible with Kubernetes versions 1.24 to 1.26.
+
+### Breaking Changes
+
+A valid license is now required for okteto to work. If you were running a previous version of okteto that has no license, an expired license or no license at all make sure to [reach out to us](https://www.okteto.com/contact/) to request one, [open a support ticket](https://okteto.com/support?_gl=1*1ikydzg*_ga*MTMyMjc0MTYyOS4xNjUyMjA1MjQ0*_ga_KSKZWJHTJZ*MTY5NTM5Njk3MS4yMTguMS4xNjk1Mzk3MDk3LjU2LjAuMA..) or signup for our [30 day free trial](https://www.okteto.com/free-trial/).
+
+### Improvements
+
+- Updates to [**Cloud Secret**](https://www.okteto.com/docs/self-hosted/administration/configuration/#cloud-provider-secret) now take immediate effect  <!-- 7032 -->
+- The Okteto CLI used for Pipeline installation has been upgraded to version [2.20.0](https://github.com/okteto/okteto/releases/tag/2.20.0)
+- Allow to configure kubetoken auth TTL through helm configuration (defaults to 24h)  <!-- 7142 -->
```

Comment:
Could you add a link to this section? https://www.okteto.com/docs/1.13/self-hosted/administration/configuration/#kubetoken